pkg/apis/apirator/v1alpha1/apimock_types.go

```diff
@@ -25,9 +25,14 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 const (
-	PROVISIONED = "PROVISIONED"
-	ERROR       = "ERROR"
-	INVALID_OAS = "INVALID_OAS"
+	PROVISIONED         = "PROVISIONED"
+	ERROR               = "ERROR"
+	InvalidOas          = "INVALID_OAS"
+	CfgMapCreated       = "ConfigMapCreated"
+	ServiceCreated      = "ServiceCreated"
+	DeploymentCreated   = "DeploymentCreated"
+	IngressEntryCreated = "IngressEntryCreated"
+	CfgMapUpdated       = "CfgMapUpdated"
```

Comment:
I believe that `InvalidOas`, `CfgMapCreated`, `ServiceCreated`, `DeploymentCreated`, `IngressEntryCreated`, `CfgMapUpdated` could be constants at `internal/steps` package.